articles/api-auth/tutorials/adoption/oidc-conformant.md

```diff
@@ -0,0 +1,56 @@
+---
+title: OIDC-conformant clients
+description: List of breaking changes for OIDC-conformant clients
+---
+
+<%= include('./_about.md') %>
+
+# OIDC-conformant clients
+
+In order to make the transition to the [OIDC-conformant authentication pipeline](/api-auth/tutorials/adoption) more predictable, clients now include an option called "OIDC Conformant", available under **Advanced Settings > OAuth**:
+
+![OIDC-conformant client setting](/media/articles/dashboard/oidc_conformant.png)
+
+For this flag to be visible, your Auth0 tenant needs to have "OAuth 2.0 API Authorization" enabled under [**Account settings > Advanced**](https://${manage_url}/#/account/advanced):
```

Comment:
I think this flag is now enabled by default (and can't be disabled) right? If so, we can probably remove this paragraph and supporting screenshot.